### PR TITLE
Update a11y, support multiline captions

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**
- Reworked image HTML to use `figure` and `figcaption` elements.
- Add support for multiline captions (originally, long captions would extend beyond the width of the image; this change restricts the caption's width to the width of the image).

### CODE REVIEW

File: src/app/public/modules/image/image.component.html [hunk 1]
```diff
@@ -1,45 +1,36 @@
...
-    [src]="imageSource"
     [alt]="imageAlt"
+    [ngClass]="{
+      'sky-image-border': showBorder
+    }"
+    [src]="imageSource"
   />
-  <div *ngIf="caption"
-    class="sky-image-caption-container"
+  <figcaption *ngIf="caption"
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/image/image.component.html]: Simplified the HTML significantly to make multiline captions work.

File: src/app/public/modules/image/image.component.spec.ts [hunk 2]
```diff
@@ -15,6 +16,10 @@ import {
   SkyImageTestComponent
 } from './fixtures/image.component.fixture';
 
+function getCaptionElement(fixture: ComponentFixture<SkyImageTestComponent>): HTMLElement {
+  return fixture.nativeElement.querySelector('.sky-image-caption');
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/image/image.component.spec.ts]: No functional changes; just some cleanup to make the tests more consistent.

File: src/app/visual/image/image-visual.component.html [hunk 3]
```diff
@@ -1,5 +1,19 @@
...
+  <div
+    class="app-screenshot"
+    id="sky-image-long-caption"
+  >
+    <sky-image
+      imageSource="~/assets/demo-image.jpg"
+      captionType="success"
+      imageAlt="image for visual test"
+      caption="This caption is very long and should wrap to the next line and never extend beyond the image's width."
+      [showCaptionPrefix]="false"
```
Comment (Blackbaud-AlexKingman) [inline src/app/visual/image/image-visual.component.html]: Check your alphabetical ordering of properties here.

File: e2e/image.e2e-spec.ts [hunk 4]
```diff
@@ -133,4 +127,20 @@ describe('Image', () => {
...
+
+  it('should match previous long caption screenshot', (done) => {
+    SkyHostBrowser.scrollTo('#sky-image-long-caption');
+
+    expect('#sky-image-long-caption').toMatchBaselineScreenshot(done, {
+      screenshotName: 'image-long-caption'
+    });
+  });
+
+  it('should match previous long caption screenshot (screen: xs)', (done) => {
```
[the remark this answers is not in the document]
Comment (Blackbaud-SteveBrush) [reply e2e/image.e2e-spec.ts]: Doh!

**Pull request merged**